### PR TITLE
Feature/rename to termius

### PR DESCRIPTION
**Pull request by Maxbey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Serverauditor ssh config utility
+# Termius ssh config utility
```
Comment (EvgeneOskin) [inline README.md]: It should be "Termius CLI"

File: README.md [hunk 2]
```diff
@@ -10,50 +10,50 @@
 
 ## Installation
 
-Serverauditor ssh config utility can be installed via [pip](http://www.pip-installer.org/en/latest/index.html):
+Termius ssh config utility can be installed via [pip](http://www.pip-installer.org/en/latest/index.html):
```
Comment (EvgeneOskin) [inline README.md]: I should be "Termius CLI"

**Review by EvgeneOskin — CHANGES_REQUESTED** [review]
Please find me notes in changes.

**Comment by Maxbey** [comment]
@EvgeneOskin Readme fixed.

**Review by EvgeneOskin — APPROVED** [review]

**Pull request merged**